server/app.py

```
@@ -8,13 +8,15 @@
 from flask_bcrypt import Bcrypt
 from itsdangerous import URLSafeTimedSerializer
 from flask_mail import Mail 
+from dotenv import load_dotenv
```

Ruff (0.12.2)

`dotenv.load_dotenv` imported but unused

Remove unused import: `dotenv.load_dotenv`

(F401)